recipe/meta.yaml

```
@@ -23,22 +24,22 @@ requirements:
   host:
     - python
     - pip
-    - snappy
+    - cramjam
   run:
-    - python
-    - snappy
+    - python >=3.8
+    - cramjam
 
 test:
   imports:
     - snappy
```

Would also suggest adding `pip check` to catch any dependency issues that might crop-up

This pairs well with the bot's dependency updates suggested above 🍷 🧀

```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
  imports:
    - snappy
```